create_magic.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ARCH=x86_64
+TYPE=qcow2
+DIR=~/build
```

Comment:
I have enabled for user to deal with qemu:///system 
like mentioned on http://wiki.libvirt.org/page/SSHPolicyKitSetup
so libvirt can access my home. (added group to that policy)